lib/rack/tracker/google_global/google_global.rb

```diff
@@ -46,6 +44,22 @@ def set_options
 
   private
 
+  def build_trackers
+    options[:trackers].map(&method(:call_tracker)).reject(&method(:invalid_tracker?))
+  end
+
+  def call_tracker tracker
+    if tracker[:id].respond_to?(:call)
+      tracker.merge(id: tracker[:id].call(env))
+    else
+      tracker
+    end
+  end
+
+  def invalid_tracker? tracker
```

Comment:
I think in the rest of the codebase we use parentheses.

Reply:
fixed.